tox.ini

```diff
@@ -11,6 +11,8 @@ deps =
     nose
     unittest2
     protobuf==3.0.0-alpha-1
+setenv =
+    PYTHONPATH = {toxinidir}/_testing
```

Comment:
See my comment below to @nathanielmanistaatgoogle. This is a discussion I've been trying to have for a week or two but not getting much traction.

If we can dive in front of both `tox` and `nosetests` (called from `tox`) to do it, I am all for it. If not, I don't think re-engineering those tools is worth the tradeoff.